app/code/Magento/Sales/Controller/Adminhtml/Order/Creditmemo/AddComment.php

```diff
@@ -43,6 +44,11 @@ class AddComment extends \Magento\Backend\App\Action
      */
     protected $resultRawFactory;
 
+    /**
+     * @var \Magento\Framework\View\Result\PageFactory
+     */
+    protected $resultPageFactory;
+
     /**
      * @param Action\Context $context
```

Comment:
By changing alias in this file you broke following import.

Reply:
Resolved